src/components/landing/Hero.js

```diff
@@ -1,16 +1,84 @@
 import Link from "next/link";
-import styles from "./hero.module.css";
+import Image from 'next/image';
+
 function Hero() {
   return (
-    <section className={styles.heroSection}>
-      <div className={styles.heroWrap}>
-        <h1>Find your people</h1>
-        <p>Find communities that interest you</p>
-        <p>Are you a community manager?</p>
-        <p>Add your community to our directory for users to find and join</p>
-        <Link href="/register" className={styles.register}>
-          Get Started
-        </Link>
+    <section className="bg-white flex flex-col justify-center">
+      <div className="flex items-center relative  my-8 mx-4">
+        <div>
+          <Image
+            src="/arrow.png"
+            className="absolute lg:left-40 lg:top-28 hidden lg:block"
+            height={107}
+            width={80}
+            alt="looping arrow pointing downwards"
+          />
+        </div>
+        <div className="flex flex-col max-w-4xl mx-auto items-center space-y-10 justify-center md:text-sm">
+          <div className="font-bold lg:text-6xl  text-center ">
+            <h1>
+              Join the <span className="text-primary bold">Tech Community</span>{" "}
+              and get to discover people from all around the world!
+            </h1>
+          </div>
+          <p className="px-4 lg:text-xl md:px-2 lg:visible text-center sm:px-20  xl:px-20">
+            Get to find different techies across the globe, build connections,
+            interactions and networks by joining the community of developers and
+            designers.
+          </p>
+          <div className="flex justify-center">
+            <Link
+              href="/register"
+              className="bg-primary rounded-lg text-white text-2xl px-10 py-3 "
+            >
+              Click to Register
```

Comment:
Let's change this button text to: "Add Your Tech Community"

Reply:
Done